capstone-app/src/app/modules/maps/map/map.component.html

```diff
@@ -14,17 +15,27 @@
 
 <div #mapContainer>
     <google-map class="map"
-        [center]="getCurrentOrSetLocation()"
-        height="750px"
-        width="100%"
-        [zoom] = "zoom"
-        (mapRightclick)="addMarker($event)">
-        <map-marker #markerp class="marker"
-            *ngFor="let marker of markers"
-            [position]="marker"
-            (mapClick)="clickMarker(markerp)"
-            (mapRightclick)="removeMarker(markerp)"
-        ></map-marker>
-        <map-info-window>Custom marker</map-info-window>
+                [center]="location"
+                height="750px"
```

Comment:
Is there a reason to use inline styling rather than in the scss?

Reply:
Yes, I really think for good coding practice and readability, all styling aspects should be in the CSS.

Reply:
Correct me if I am wrong Sofia, but this is a strange exception to the rule because the google-map component actually takes in height and width inputs on the component. So this should be correct.
https://github.com/angular/components/blob/master/src/google-maps/google-map/google-map.ts#L81

Reply:
yes Nick map can only take the height and width inputs from the component

Reply:
Ahh, ok, very interesting. Sorry, I never knew that!